tests/nn/data_parallel/test_fsdp_grad_scaler.py

```diff
@@ -0,0 +1,41 @@
+import os
+from unittest import mock
+
+import pytest
+import torch
+from torch.cuda.amp import autocast
+import torch.nn as nn
+import torch.nn.functional as F
+
+from fairscale.nn import FullyShardedDataParallel
+from fairscale.optim.grad_scaler import ShardedGradScaler
+from fairscale.utils.testing import skip_if_no_cuda, torch_version
+
+
+@mock.patch.dict(os.environ, {"MASTER_ADDR": "localhost", "MASTER_PORT": "1337"}, clear=True)
+@skip_if_no_cuda
+def test_scaler_cpu_offload():
```

Comment:
```suggestion
def test_scaler_cpu_offload_breaks():
```

Otherwise reader doesn't know we need to fix until line 38 :)